exampleCourse/questions/demoAutograderNumpy/tests/setup_code.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import numpy.linalg as la
+
+# set up parameters
+n = np.random.randint(4, 16)
+A = np.eye(n)
```

Comment:
Why is this always the identity? Seems like it might allow bad student solutions to be graded as correct? Perhaps we should use a random matrix that is invertible? For example, a random positive diagonal plus small off-diagonal elements, so the matrix is diagonally dominant and thus invertible?

Reply:
Yes, I agree that this wouldn't be a great "real" question 🙂 

I'll change this to be a random full-rank matrix so that it'll be a better example

Reply:
Awesome! I think it's good to have these demo questions be as real as possible, in case people copy them, but without being overly complex (can be hard! 😄).